lib/src/models/multireddit.dart

```diff
@@ -241,8 +244,23 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
           .replaceAll(_multiredditRegExp, name)
           .replaceAll(_userRegExp, user);
 
+  // Converts input text into camelCase.
+  static String _convertToCamelCase(String str) {
+    if (str == null) {
+      return null;
+    }
+    final RegExp _invalidRegExp = new RegExp(r'(\s|\W|_)+');
```

Comment:
Did you get this from somewhere? This matches white space, a word character (A-Z, a-z, 0-9, _), or "\_"? Basically, if you could walk me through what's going on here with the regexp that would help me understand :)

Reply:
You're right, I copied this from somewhere else, will fix!

Reply:
Fixed.